ush/python/pygfs/task/aero_emissions.py

```diff
@@ -37,48 +38,363 @@ def __init__(self, config: Dict[str, Any]) -> None:
         """
         super().__init__(config)
 
-        local_variable = "something"
+        nforecast_hours = self.task_config["FHMAX_GFS"]
+        blend_start_date = self.task_config["PDY"]
+        blend_end_date = blend_start_date + to_timedelta(f'{nforecast_hours + 24}H')
+        forecast_dates = list(rrule(freq=DAILY, dtstart=blend_start_date, until=blend_end_date))
+
+        # add forecast_dates to the task_config for parsing yaml file
+        localdict = AttrDict({"forecast_dates": forecast_dates})
+        self.task_config = AttrDict(**self.task_config, **localdict)
+
+        # populate yaml file and add to task_config
+        logger.info(f"Read the prep_emission configuration yaml file {self.task_config.PREP_EMISSION_CONFIG}")
+        self.task_config.aero_emission_yaml = parse_j2yaml(self.task_config.PREP_EMISSION_CONFIG, self.task_config)
+        logger.debug(f"aero_emission_yaml:\n{pformat(self.task_config.aero_emission_yaml)}")
+
+        config = self.task_config.aero_emission_yaml['aero_emissions']['config']
+        emission_files = {}
+        for emission_type in ['qfed', 'hfed', 'gbbepx', 'climo']:
+            emission_files[emission_type] = [
+                os.path.basename(src)
+                for src, _ in config['data_in'][emission_type]['copy']
+            ]
+        n_persist = config['n_persist']
 
         localdict = AttrDict(
-            {'variable_used_repeatedly': local_variable}
+            {
+                "cdate": blend_start_date,
+                "nforecast_days": nforecast_hours // 24,
+                "workdir": self.task_config.DATA,
+                "current_date": self.task_config.PDY,
+                'config': config,
+                'emistype': config['emistype'],
+                'emisfiles': emission_files,
+                'n_persist': n_persist
+            }
         )
 
+        # TODO: if AERO_EMIS_FIRE is not 'blended' we don't want to do anything!
```

Comment:
@WalterKolczynski-NOAA could you share some insights on how we should accomplish this? Is there a way to get the value of `AERO_EMIS_FIRE` (from `parm/config/{gefs,gfs)/config.aero` within this task? Or should the Task just be skipped somewhere further up the workflow scripting chain?

cc: @bbakernoaa

Reply:
If the job isn't needed at all, then it should be skipped at the workflow level (in `gfs_tasks.py`/`gefs_tasks.py`).

Reply:
Sorry, it should be removed in `workflow/applications/gfs_cycled.py` etc, which may also require changed to `gfs_tasks.py`, etc.